cypress/integration/application/index.js

```diff
@@ -1,11 +1,14 @@
-import config from '../../../src/app/lib/config/services';
-import { describeForLocalOnly } from '../../support/limitEnvRuns';
+import config from '../../support/config/services';
 
-describeForLocalOnly('Application', () => {
+describe('Application', () => {
   Object.keys(config)
-    .filter(service => service !== 'default')
+    .filter(service => service !== 'news')
+    .filter(service =>
+      Object.keys(config[service].pageTypes).some(
+        pageType => config[service].pageTypes[pageType].path !== undefined,
```

Comment:
Can we reuse `serviceHasPageType` https://github.com/bbc/simorgh/pull/3053/files#diff-a372023abbaa311428537c61de200162R4?